backend/webapi/Features/Parties/ProfileStatus.Model.cs

```diff
@@ -273,6 +273,24 @@ _ when (profile.EndorsementPlrStanding.HasGoodStanding
                 };
             }
         }
+        public class ImmsBCSection : ProfileSection
+        {
+            internal override string SectionName => "immsbc";
+
+            protected override StatusCode Compute(ProfileData profile)
+            {
+                return profile switch
+                {
+                    _ when (profile.EndorsementPlrStanding.HasGoodStanding
+                        || profile.PartyPlrStanding
+                               .With(ImmsBC.AllowedIdentifierTypes)
+                            .HasGoodStanding)
+                        && profile.HasBCProviderCredential => StatusCode.Complete,
+                    { HasBCServicesCardCredential: true } => StatusCode.Incomplete,
+                    _ => StatusCode.Locked
```

Comment:
Please double check your logic here. I assume you based this off the `ProvincialAttachmentSystemSection`; but that section functions differently than all of our other ones. For PAS, we mark it as "complete" once the user has met all the requirements to enter PAS since there isn't actually a PAS enrolment for them to request. As well, this mentions endorsements which is not relevant to ImmsBC.

I would recommend you look at `PrescriptionRefillEformsSection` instead. For example, the section should be marked as complete when the user has an enrolment of the correct type.